pkg/JsonResource.go

```diff
@@ -56,7 +57,15 @@ func (jr *JSONResource) Load() ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	rs, err := ParseJSONRuleset(data)
+	firstRune := string(bytes.TrimSpace(data)[0])
+
+	var rs string
+
+	if firstRune == "[" {
+		rs, err = ParseJSONRuleset(data)
+	} else if firstRune == "{" {
+		rs, err = ParseJSONRule(data)
+	}
```

Comment:
```suggestion
	} else {
		err = errors.New("invalid JSON input")
	}
```
Looks good! We just need to catch the extra case where we may have loaded something which is not valid JSON.